src/Orchard/Environment/Extensions/Folders/IExtensionFolders.cs

```diff
@@ -4,5 +4,6 @@
 namespace Orchard.Environment.Extensions.Folders {
     public interface IExtensionFolders {
         IEnumerable<ExtensionDescriptor> AvailableExtensions();
+        IEnumerable<string> Paths { get; }

```

Comment:
I'm under the impression you replied to this comment of mine: https://github.com/OrchardCMS/Orchard/pull/5604/files#r36408832

I think that such paths (if we're talking about path prefixes where actual extensions were loaded from) should be store on an app-level (not tenant-level): it's easier and since we don't talk about tens of thousands of possible locations, all different per tenants here I don't see much value in maintaining such a list for faster lookup (my impression is that this was the aim).